commands/commands.go

```diff
@@ -224,7 +223,7 @@ func determineIncludeExcludePaths(includeArg, excludeArg string) (include, exclu
 
 func printHelp(commandName string) {
 	if txt, ok := ManPages[commandName]; ok {
-		fmt.Fprintf(os.Stderr, txt)
+		fmt.Fprintf(os.Stderr, "%s\n", strings.TrimSpace(txt))
```

Comment:
Hmm, this puts the command snug under the prompt which I find harder to read, e.g.:
```
steve@Stevens-MacBook-Pro ~/p/G/s/g/g/git-lfs> git lfs pull --help
git lfs pull [options] [<remote>]

Download Git LFS objects for the currently checked out ref, and update
```
I find this harder to read than
```
steve@Stevens-MacBook-Pro ~/p/G/s/g/g/git-lfs> git lfs pull --help

git lfs pull [options] [<remote>]

Download Git LFS objects for the currently checked out ref, and update
```